### PR TITLE
Stage to Main

### PR DESCRIPTION
**Pull request by JingleH**
Everything on our last stage->main was reverted in a hotfix. This is the retry after removing PRs that could be causing issues.

Test URLs:
- Before: https://main--express--adobecom.hlx.live/express/templates/?lighthouse=on
- After: https://stage--express--adobecom.hlx.live/express/templates/?lighthouse=on

### CODE REVIEW

**Review by echen-adobe — DISMISSED** [review]

**Review by echen-adobe — DISMISSED** [review]

File: express/blocks/frictionless-quick-action/frictionless-quick-action.css [hunk 1]
```diff
@@ -137,7 +136,8 @@
 }
 
 .frictionless-quick-action.block > div.hidden {
-    display: none;
+    opacity: 0;
```
Comment (fullcolorcoder) [inline express/blocks/frictionless-quick-action/frictionless-quick-action.css:139]: Would visibility:hidden be better here? Opacity 0 allows users to interact with an element.

File: express/scripts/instrument.js [hunk 2]
```diff
@@ -711,6 +711,10 @@ function martechLoadedCB() {
   // Fire the landing:viewedPage event
   sendEventToAdobeAnaltics('landing:viewedPage');
 
+  // if (getMetadata('quickaction-upload-page') === 'on') {
```
Comment (fullcolorcoder) [inline express/scripts/instrument.js:714]: Is commented code in here intentional?

File: express/blocks/frictionless-quick-action/frictionless-quick-action.css (hunk 1 above)
Comment (JingleH) [reply express/blocks/frictionless-quick-action/frictionless-quick-action.css]: good idea! I've changed it to visibility hidden. This overlay is technically no longer needed, but to ship it asap let's keep it here and remove it in the next release

File: express/scripts/instrument.js (hunk 2 above)
Comment (JingleH) [reply express/scripts/instrument.js]: this was commented out in another last-minute discussion. It's removed. Thanks!

**Review by apganapa-adobe — APPROVED** [review]

**Review by echen-adobe — APPROVED** [review]

**Review by fullcolorcoder — APPROVED** [review]

**Pull request merged**